annonatate/utils/github.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import os, json
+import base64
+from flask_github import GitHub
+
+import pdb
+
+class GitHubAnno(GitHub):
+  def setbork(self):
+    self.bork = 'bork'
```

Comment:
What is this? Are these necessary lines?

Reply:
sorry, just diagnostics while I was making sure the new GitHubAnno class was being used. I've deleted them in my second commit.